perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -126,8 +126,10 @@ sub report_new_ajax : Path('mobile') : Args(0) {
 
     # Apps are sending email as username
     # Prepare for when they upgrade
-    if (!$c->get_param('username')) {
-        $c->set_param('username', $c->get_param('email'));
+    my $username_field = ( $c->get_param('submit_sign_in') || $c->get_param('password_sign_in') )
+        ? 'username': 'username_register';
```

Comment:
username_register is confusing as I expect that to be the one to use when they are registering which is the opposite of the case. maybe username_logged_in?

Reply:
It matches password_register? It is the one they are using when they are registering (whilst making a new report).

Reply:
It's also what I see when I'm already logged in though which is what's confusing me.

Reply:
Have set it back to username for logged in form.